src/io/scratchBT.js

```diff
@@ -0,0 +1,37 @@
+const JSONRPCWebSocket = require('../util/jsonrpc');
+
+const ScratchLinkWebSocket = 'ws://localhost:20110/scratch/bt';
+
+class ScratchBT extends JSONRPCWebSocket {
```

Comment:
Presumably, this file will fill out a bit once we work on a Bluetooth Classic device extension, like EV3 which is coming up